web/menu.jspf

```diff
@@ -71,24 +71,153 @@ org.opensolaris.opengrok.web.Util"
             pMap.put(name, esc);
         }
 %>
+<script type="text/javascript" src="<%= request.getContextPath() %>/js/jquery-3.2.0.min.js"></script>
 <script type="text/javascript">/* <![CDATA[ */
-    document.domReady.push(function() { domReadyMenu(); });
+    document.xrefPath = '<%= request.getContextPath() + Prefix.XREF_P
+    %>';document.domReady.push(function() { domReadyMenu(); });
 /* ]]> */
+var initialWidth = null;
+var isFirst = true;
+var checkWidth = function(){
+    var width = (window.innerWidth > 0) ? window.innerWidth : screen.width;
+	console.log(width);	
+	if(width <= 770) {
```

Comment:
the width is not really important to be configurable, rather the value if using the "old" and "new" design is important